getting-started/setup-multi-node-basic.md

```diff
@@ -23,30 +24,70 @@ node, and follow the instructions in the next section.  More information will be
 forthcoming as TimescaleDB multi-node is made available on these cloud platforms.
 
 ## Initialize data nodes from the access node [](init_data_nodes_on_access_node)
-While connected to the access node (psql), use the command:
+
+Once logged in on the access node, it is necessary to add data nodes
+to the local database before creating distributed hypertables. This
+will make the data node available for use by distributed hypertables
+and the access node will also connect to the data node and initialize
+it.
+
+While connected to the access node as a superuser (e.g., via `psql`),
+use the command:
 
 ```sql
 SELECT add_data_node('example_node_name', host => 'example_host_address');
 ```
 
-`example_node_name` should be a unique name for the node. `example_host_address` 
-is the host name or IP address of the data node.
+`example_node_name` should be a unique name for the
+node. `example_host_address` is the host name or IP address of the
+data node. It is also possible to specify a password to authenticate
+with using the optional `password` parameter. But this is only
+necessary if password authentication is used to connect to data nodes
+and the password is not provided through other means (e.g., a local
+password file). See the [`add_data_node`][add_data_node] API reference
+documentation for detailed information about this command.
+
+It should now be possible to create distributed hypertables. Note,
+however, that in order to create and use distributed hypertables as a
+non-superuser, the user role needs to be distributed as well (i.e.,
+exist on all data nodes) and requires privileges to use the data nodes
+on the access node. Please refer to the next section for instruction
+on how to create distributed roles and grant data node privileges.
 
-The system should be up and running at this point.
+## (Optional) Add distributed user roles
```

Comment:
The notion of a "distributed role" suggest that it is somehow maintained and is a novel concept. This section only talks about how to add roles on all nodes of a distributed database.
```suggestion
## (Optional) Adding roles for distributed hypertables
```

Reply:
"Distributed" is used as adjective here, and I think users know what it means for something to be "distributed".  I don't think it is a bad idea to introduce the concept of distributed roles or, more generally, distributed objects. Otherwise, it becomes very cumbersome to always say "roles that exist on all data nodes". 

I will try to be more "general" for now, but I think we need to figure out the proper terminology to use throughout the documentation.